internal/host/statemachine.go

```diff
@@ -438,5 +438,17 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		})
 	}
 
+	// Noop transitions for day2 hosts
+	for _, state := range []stateswitch.State{
+		stateswitch.State(models.HostStatusAddedToExistingCluster),
```

Comment:
Isn't the ticket talked about installed state? or did i miss something?

Reply:
the installed status for day2 on cloud is AddedToExistingCluster